plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -344,12 +344,17 @@ def onUpdateScribbles(self):
             self.updateServerSettings()
             self.reportProgress(60)
 
+            roiNode = slicer.mrmlScene.GetFirstNodeByClass("vtkMRMLAnnotationROINode")
```

Comment:
Assuming first node of type ROI?  Why don't u have a drop-down in the scribbles.. that can list all the ROI nodes.. and by default first node is selected.. in case user has added multiple ROI nodes.. he has the choice to fix/correct the selection while running the scribbles..

Reply:
Yes, thats right! This currently only looks for the first ROI node. The reason for this choice was primarily to keep the code simple while improving the interaction with the usage of ROI and secondly to avoid further cluttering the UI. At the moment the purpose of ROI is only to avoid having the user giving a lot of background scribbles for building a good likelihood model.

It certainly is doable to add additional ROI and have user select the right one. I may need to look into this further and come up with the correct way of highlighting the selected ROI as well during selection.  The rest of the backend logic will remain same.

Reply:
that's fine... you can make it as selectable list in UI later

Reply:
Many thanks! Will add this to my backlog and revisit soon.